blog/2023-10-20-rebranding.mdx

```diff
@@ -0,0 +1,57 @@
+---
+slug: tolgee-rebranding
+title: "Tolgee's Exciting Transformation: A Modern, Fresh Look"
+description: Tolgee rebranding to new colors. We choose pink for it's modern and vibrant tone.
+authors: [sgranat]
+tags: ['brand', 'pink', 'rebranding', 'tolgee']
+---
+
+import { ReviewAuthor } from '../platform/shared/_ReviewAuthor';
+import { ScreenshotWrapper } from '../platform/shared/_ScreenshotWrapper';
+
+We're thrilled to introduce our refreshed brand, including our logo in a vibrant shade of pink, and a modern, user-friendly design for our app and website. We've heard your feedback, and we've taken it to heart.
+
+![New Tolgee landing page](/img/blog/rebranding/tolgee-landing-page.png)
+
+
+<!-- truncate-->
+
+## A New Look for a Bright Future
+
+Our journey to reimagining Tolgee began with a critical examination of our interface, and the consensus was clear: our old toolbar and color scheme just didn't cut it. So, we took the opportunity to overhaul the entire application's design, creating a cohesive and contemporary user experience.
+
+<ReviewAuthor
+  author={{
+    imageURL: '/img/blog/profile-photos/placeholder.png',
+    name: 'Martin P.',
+    title: 'Freelancer',
+  }}
+  description="The application doesn't have any bigger issues, just the design of UI is a little old-school."
+  capterraReview="https://www.capterra.com/p/10002120/Tolgee/reviews/4342787/"
+/>  
+
+Your feedback is invaluable, and we want to thank our users for their insights. The transformation of Tolgee is a direct response to your needs, and we hope it reflects our commitment to making your localization projects smoother and more enjoyable.
+
+### Unifying Our Brand Colors
+
+One of the key aspects of our rebranding was unifying our brand colors as until now we've been using different colors in light and dark mode. We've carefully chosen a palette that's harmonious and inviting, ensuring that every interaction with our platform is both visually pleasing and user-friendly.
+
+### Matching Dark Mode and Light Mode
+
+We understand the importance of choice, so we've also matched our Dark Mode to the sophistication and modern appeal of our Light Mode. Whether you prefer light or dark, Tolgee is now visually consistent, offering a seamless experience.
+
+
+**Old Toolbar - The Starting Point**
+<ScreenshotWrapper src="/img/blog/rebranding/tolgee-old.png" alt="Tolgee old look" />
+
+**Fresh Design - A Modern Makeover**
+<ScreenshotWrapper src="/img/blog/rebranding/tolgee-new.png" alt="Tolgee new look" />
+
+**Matching dark mode**
+<ScreenshotWrapper src="/img/blog/rebranding/tolgee-new-dark.png" alt="Tolgee dark mode" />
+
+## Join the New Tolgee Experience
+
+We invite you to explore the reimagined Tolgee. From our stunning new logo to the revitalized application, we're excited to serve you better than ever. We're confident that this transformation will enhance your software localization journey and make it even more enjoyable.
```

Comment:
I'd rewrite it as follows: Come and check out the all-new Tolgee! We've got a fresh logo and a revamped app, and we're thrilled to provide you with an even better experience. We believe these changes will make your software localization journey even more enjoyable! Let us know how you like it in the [Tolgee Slack community](https://Tolg.ee/slack)